src/wp-includes/comment-template.php

```diff
@@ -1707,6 +1707,7 @@ function get_comment_reply_link( $args = array(), $comment = null, $post = null
 			'postid'         => $post->ID,
 			'belowelement'   => $args['add_below'] . '-' . $comment->comment_ID,
 			'respondelement' => $args['respond_id'],
+			'replyto'        => esc_attr( sprintf( $args['reply_to_text'], $comment->comment_author ) ),
```

Comment:
No need to `esc_attr()` this, it will be escaped in when setting `$data_attribute_string` below.

```suggestion
			'replyto'        => sprintf( $args['reply_to_text'], $comment->comment_author ),
```